test/cassandart_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
 
     setUpAll(() async {
       client = await Cluster.connect(
-        ['localhost:9042'],
+        ['remote-cassandra-server:9042'],
```

Comment:
Let's keep it `localhost`, or better: make it configurable, defaulting to `localhost`